src/IssueViz.UnitTests/Editor/DocumentNavigatorTests.cs

```diff
@@ -42,14 +43,13 @@ public class DocumentNavigatorTests
 
         private Mock<IOutliningManagerService> outliningManagerServiceMock;
         private Mock<IEditorOperations> editorOperationsMock;
+        private Mock<IThreadHandling> threadHandlingMock;
```

Comment:
I tried staying consitent with how the tests were already written instead of changing everything which is why its here